lib/amqp.js

```diff
@@ -69,10 +90,10 @@ class Amqp {
 
             let decryptedContent;
             try {
-                decryptedContent = encryptor.decryptMessageContent(message.content, message.properties.headers);
+                decryptedContent = await self.decryptMessage(message);
```

Comment:
It's not the best idea to use the same method name twice.

Reply:
Agree.